forest_example.ipynb

```diff
@@ -314,7 +314,7 @@
    "source": [
     "# or you can simply use the predict method\n",
     "\n",
-    "y_pred = clf.predict(X_test)\n",
+    "y_pred = clf.predict(X_test).astype(int)\n",
```

Comment:
Is this now necessary?

Reply:
See commentary below as both are related.